### ISSUE TITLE
📖Add a section for Kubevirt in the Quick Start doc

### ISSUE THREAD

**Issue by sayantani11** [kubernetes-sigs/cluster-api#5845]
**User Story**
As we know, we now have Cluster API Provider for Kubevirt, and recently a issue was raised in Kubevirt about how a quickstart guide/example was missing for it.
https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt/issues/36

**Detailed Description**
It will be great if we could have a qucikstart section for Kubevirt in the Cluster API Quick Start doc, makes it easier for a user/contributor who are new to kubevirt to get started.

**Anything else you would like to add:**

[Miscellaneous information that will assist in solving the issue.]

/kind feature

**Comment by fabriziopandini** [comment]
@sayantani11 go for it! feel free to send a PR proposing changes to the quick start page (prior art https://github.com/kubernetes-sigs/cluster-api/pull/3827)

/remove-kind proposal
/kind documentation
/milestone v1.1

**Comment by fredgate** [comment]
I discovered this provider by chance.  
It should also be added to the list of providers : https://cluster-api.sigs.k8s.io/reference/providers.html

**Comment by sayantani11** [comment]
@fabriziopandini This can be done after KubeVirt  has its first release

**Comment by sayantani11** [comment]
> I discovered this provider by chance. It should also be added to the list of providers : https://cluster-api.sigs.k8s.io/reference/providers.html

Okay will add a new issue for it

**Comment by fabriziopandini** [comment]
> @fabriziopandini This can be done after KubeVirt has its first release

clusterctl allows to install prerelease as well (https://cluster-api.sigs.k8s.io/clusterctl/commands/init.html#provider-version), but at least a release or a pre-release should exist

**Comment by sayantani11** [comment]
> > @fabriziopandini This can be done after KubeVirt has its first release
> 
> clusterctl allows to install prerelease as well (https://cluster-api.sigs.k8s.io/clusterctl/commands/init.html#provider-version), but at least a release or a pre-release should exist

I don't think that KubeVirt even has a pre-release as of now. They are still working on a lot of things as much as communicated via Slack, but yeah probably soon

**Comment by fabriziopandini** [comment]
@sayantani11 any update on this issue?
